components/power_agent/actuator.py

```
@@ -0,0 +1,962 @@
+            self._groups.clear()
+            # Invalidate the cross-library identity map. DCGM may
+            # re-enumerate after the hostengine restart (different
+            # gpuId ordering if the operator changed visibility), so
+            # we must rebuild on next `list_running_pids` call.
+            self._dcgm_uuid_by_idx = None
+            self._nvml_index_by_uuid = None
+            try:
+                if self._handle is not None:
+                    self._handle.Shutdown()
+            except Exception:
+                pass
```

Ruff (0.15.13)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)